scalafx/src/main/scala/scalafx/application/Platform.scala

```diff
@@ -41,6 +41,20 @@ object Platform {
     jfxa.Platform.exit()
   }
 
+  /**
+   * Requests the Java Runtime to perform a pulse. This will run a pulse
+   * even if there are no animation timers, scene graph modifications,
+   * or window events that would otherwise cause the pulse to run.
+   * If no pulse is in progress, then one will be scheduled to
+   * run the next time the pulse timer fires.
+   * If there is already a pulse running, then
+   * at least one more pulse after the current pulse will be scheduled.
+   * This method may be called on any thread.
+   *
+   * @since 9
+   */
+  def requestNextPulse: Unit = jfxa.Platform.requestNextPulse
```

Comment:
The method updates state so it should be declared with parentheses: `def requestNextPulse()`